src/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,143 @@
+import { CaretRight, DotsThree } from '@phosphor-icons/react';
+import { forwardRef, FunctionComponent, ReactNode, SVGProps } from 'react';
+import { Dispatch } from 'redux';
+import Dropdown from '../dropdown/Dropdown';
+import { DropTargetMonitor } from 'react-dnd';
+import BreadcrumbsItem, { BreadcrumbItemData, BreadcrumbsMenuProps } from './BreadcrumbsItem';
+
+export interface BreadcrumbsProps<T extends Dispatch> {
```

Comment:
Can we add a description like the one in the Base Dialog component for each parameter (JSDoc)?

Reply:
Sure!